knowledge-base/kb-restrictfirstaxis-to-chartpanewidth.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How To Restrict the First Axis Label to be Within the Chart Pane Width
```

Comment:
You can use RadChartView here instead of Chart, so you can have bigger variety of used words.